src/main/resources/assets/minecolonies/lang/en_us.json

```diff
@@ -104,15 +98,15 @@
   "minecolonies.config.allowplayerschematics": "Player Schematics",
   "minecolonies.config.allowplayerschematics.comment": "Should player-made schematics be allowed",
   "minecolonies.config.allowglobalnamechanges": "Name Changes",
-  "minecolonies.config.allowglobalnamechanges.comment": "Should players be allowed to change names? -1 for false, 0 for specific groups, 1 for true.",
+  "minecolonies.config.allowglobalnamechanges.comment": "Should players be allowed to change citizen names? -1 for false, 0 for certain players only, 1 for true. (To set which players can change the names, see \"Special Permissions Group\" below.)",
   "minecolonies.config.holidayfeatures": "Holiday Features",
-  "minecolonies.config.holidayfeatures.comment": "Should special holiday content be displayed?",
-  "minecolonies.config.updaterate": "Update-Rate",
+  "minecolonies.config.holidayfeatures.comment": "Should special holiday content be displayed? (Santa hats around Christmas time, semi-transparent citizens near Halloween, etc.)",
+  "minecolonies.config.updaterate": "Update Rate",
   "minecolonies.config.updaterate.comment": "AI Update rate. Increase this to improve performance. This throttles citizen updates.",
-  "minecolonies.config.dirtfromcompost": "Compost->Dirt",
+  "minecolonies.config.dirtfromcompost": "Compost to Dirt",
   "minecolonies.config.dirtfromcompost.comment": "Quantity of dirt per compost filling.",
-  "minecolonies.config.luckyblockchance": "Lucky-Blocks",
-  "minecolonies.config.luckyblockchance.comment": "Chance to get a lucky block as a percentage.",
+  "minecolonies.config.luckyblockchance": "Ore Chance",
+  "minecolonies.config.luckyblockchance.comment": "Chance for the Miner to get an ore when mining cobblestone or stone. This is a percentage.",
```

Comment:
Maybe emphasize that it's a bonus ore?  I dun't expect it to happen often, but could see someone think this limits even the miner's ability to gather worldgen ores.